xml/security_pcidss.xml

```diff
@@ -7,272 +7,1251 @@
+    <variablelist>
+     <varlistentry>
+      <term>
+       6.2.a Examine policies and procedures related to security patch
+       installation to verify processes are defined for: (1) Installation of
+       applicable critical vendor-supplied security patches within one month
+       of release, (2) Installation of all applicable vendor-supplied
+       security patches within an appropriate time frame (for example, within
+       three months).
+      </term>
+      <listitem>
+       <para>
+        To identify patches that
+        need to be installed to secure your system, do the following:
+       </para>
+       <!-- FIXME: procedure -->
+       <para>
+        First refresh all software repositories so that we have up-to-date information.
+       </para>
+       <screen>zypper refresh</screen>
+       <para>
+        Now we can look for important security fixes that have not yet been
+        installed:
+       </para>
+       <screen>zypper list-patches --category security --severity important</screen>
+       <para>
+        It is also possible to search for CVE or SUSE Bugzilla numbers. By
+        default only needed patches are listed by this command. To show also
+        patches that have already been installed use <option>--all</option>:
+       </para>
+       <screen>zypper list-patches --all --cve=CVE-2016-4957</screen>
+       <para>
+        Details of individual patches can be listed with the patch-info
+        subcommand:
+       </para>
+       <screen>zypper patch-info SUSE-SLE-SERVER-12-SP1-2016-600</screen>
+       <para>
+        To install only important security patches use the patch command. All
+        <command>zypper</command> commands can be run with the <option>--non-interactive</option> to run
+        automatically<!--fixme: wording-->:
+       </para>
+       <screen>zypper patch --category security --severity important</screen>
+      </listitem>
+     </varlistentry>
+    </variablelist>
+   </sect3>
+  </sect2>
+  <sect2>
+   <title>Implement Strong Access Control Measures</title>
+   <para/>
+   <sect3>
+    <title>Requirement 7: Restrict access to cardholder data by business need to know</title>
+    <para>Summary:</para>
+    <para>
+     Access control is a complex topic if it comes to the operating system.
+     Again, the &pcidssa; requirement is quite fuzzy at this point and does not
+     specifically state to which degree the restriction needs to be fulfilled.
+     &productname; comes with all general Linux tools to limit and
+     restrict access to certain system areas and components.
+    </para>
+    <para>
+     Access can be controlled via specific users and groups of users. This
+     are the traditional Unix permission settings. A more flexible mechanism
+     for file systems are Access Control Lists (ACLs) that offer a more
+     graduated approach. &selnx; can be used to separate the system to a
+     maximum and to protect processes from gaining more resources and access
```

Comment:
<del>can be used to separate the system to a maximum</del> for maximum system separation?